src/main/kotlin/no/skatteetaten/aurora/gobo/resolvers/application/ApplicationResolver.kt

```diff
@@ -19,8 +19,15 @@ class ApplicationResolver(private val registryMetadataResolver: RegistryMetadata
         get(): ImageRepository? = this.applicationDeployments.asSequence()
             .mapNotNull { it.dockerImageRepo }
             .map { ImageRepository.fromRepoString(it) }
-            .firstOrNull {
-                val metadata = registryMetadataResolver.getMetadataForRegistry(it.registryUrl)
-                !metadata.isInternal
-            }
+            .firstOrNull { !dockerRegistry.isInternal(it.registryUrl) }
+}
+
+@Component
+class DockerRegistry(@Value("\${integrations.internal-registry.url:docker-registry.default.svc:5000}") val internalRegistryAddress: String) {
```

Comment:
Jeg trodde dette skulle bort fra gobo?

Reply:
Vi fant problemer med å fjerne deler av dette, for å fjerne det krever det litt større endringer i ImageRepository og kanskje et nytt endepunkt i Cantus. Så dette ble for stort for scopet til oppgaven. Det har blitt laget en oppgave på det som skal diskuteres neste teammøte